static/bidder-info/ownadx.yaml

```diff
@@ -0,0 +1,14 @@
+#endpoint: "http://{{.Host}}-prebid.owadx.com/?seat={{.AccountID}}&token={{.SourceId}}"
+endpoint: "http://{{.Host}}/bidder/bid/{{.AccountID}}/000?token={{.SourceId}}"
+#endpoint: "http://beta-bidder.vrtzads.com/vzbidder/bid?exchangeId=705712f550cc479497fbdbf2edce5569&host={{.Host}}&seat={{.AccountID}}&token={{.SourceId}}"
```

Comment:
PTAL at bidders with dynamic hosts https://github.com/prebid/prebid-server/issues/2612 issue. As per guidelines, bidders must not have dynamic domain name

Reply:
Resolved .change dynamic domain name